crates/admin/src/cluster_controller/service/state.rs

```diff
@@ -230,7 +230,7 @@ where
                 _ = self.find_logs_tail_interval.tick() => {
                     self.logs_controller.find_logs_tail();
                 }
-                _ = OptionFuture::from(self.log_trim_check_interval.as_mut().map(|interval| interval.tick())) => {
+                Some(_) = OptionFuture::from(self.log_trim_check_interval.as_mut().map(|interval| interval.tick())) => {
```

Comment:
Thanks for fixing this 🙏 I wasn't fully aware of the `OptionFuture` API back when I wrote it.